docs/api/unified_api/add_to_cart_product_chosen_by_product_options.rst

```diff
@@ -0,0 +1,84 @@
+Add to cart product chosen by product options
```

Comment:
How to add product variants to the cart in Sylius API?

Reply:
How to add product variants by options to the cart in Sylius API?